io/plugins/eu.esdihumboldt.hale.io.shp.ui/src/eu/esdihumboldt/hale/io/shp/ui/SimpleTargetCRSConfigurationPage.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright (c) 2020 wetransform GmbH
+ * 
+ * All rights reserved. This program and the accompanying materials are made
+ * available under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * Contributors:
+ *     wetransform GmbH <http://www.wetransform.to>
+ */
+
+package eu.esdihumboldt.hale.io.shp.ui;
+
+import org.eclipse.jface.dialogs.Dialog;
+import org.eclipse.jface.dialogs.DialogPage;
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.jface.layout.GridLayoutFactory;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Group;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.ui.PlatformUI;
+import org.geotools.referencing.CRS;
+
+import eu.esdihumboldt.hale.common.core.io.IOProvider;
+import eu.esdihumboldt.hale.common.instance.io.GeoInstanceWriter;
+import eu.esdihumboldt.hale.common.schema.geometry.CRSDefinition;
+import eu.esdihumboldt.hale.ui.io.IOWizard;
+import eu.esdihumboldt.hale.ui.io.config.AbstractConfigurationPage;
+import eu.esdihumboldt.hale.ui.io.instance.crs.SelectCRSDialog;
+
+/**
+ * Configuration page for the target CRS.
+ * 
+ * @param <W> the concrete I/O wizard type
+ * @param <P> the {@link IOProvider} type used in the wizard
+ * 
+ * @author Simon Templer
+ */
+public class SimpleTargetCRSConfigurationPage<P extends GeoInstanceWriter, W extends IOWizard<P>>
```

Comment:
Instead of copying the whole class I would prefer an approach that does not duplicate so much code.

This seems to be the main change:
![image](https://github.com/halestudio/hale/assets/375475/aa1de73a-fdd9-44ac-8bb3-2511a39f9afc)

What about adding an optional constructor argument to the original class that either gets a boolean if the message should be shown or a string for the message text and depending on the value shows the message.
Then this class here would just implement a default constructor that calls the super constructor with the respective parameter.